### PR TITLE
Add sensitivity analysis server to get default results threshold value

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/gateway/endpoints/SensitivityAnalysisServer.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.gridsuite.gateway.ServiceURIsConfig;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@Component(value = SensitivityAnalysisServer.ENDPOINT_NAME)
+public class SensitivityAnalysisServer implements EndPointElementServer {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/gateway/endpoints/SensitivityAnalysisServer.java:18]: EndPointElementServer ?

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/gateway/endpoints/SensitivityAnalysisServer.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/gateway/endpoints/SensitivityAnalysisServer.java]: Changed

File: src/main/java/org/gridsuite/gateway/endpoints/EndPointServer.java [hunk 2]
```diff
@@ -28,6 +30,10 @@ default Buildable<Route> getRoute(@NonNull PredicateSpec p) {
             .uri(getEndpointBaseUri());
     }
 
+    default Set<String> getUncontrolledRootPaths() {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/gateway/endpoints/EndPointServer.java:33]: Remove it

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/gateway/endpoints/EndPointServer.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/gateway/endpoints/EndPointServer.java]: done

File: src/test/java/org/gridsuite/gateway/ElementAccessControlTest.java [hunk 3]
```diff
@@ -149,6 +150,14 @@ public void testWithNoControl() {
             .header("Authorization", "Bearer " + tokenUser1)
             .exchange()
             .expectStatus().isOk();
+
+        // No control for some sensitivity analysis server
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/gateway/ElementAccessControlTest.java:154]: Remove it

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/test/java/org/gridsuite/gateway/ElementAccessControlTest.java (hunk 3 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/gateway/ElementAccessControlTest.java:154]: just the line comment ?

Comment (SlimaneAmar) [reply src/test/java/org/gridsuite/gateway/ElementAccessControlTest.java:154]: No the test

Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/gateway/ElementAccessControlTest.java]: Done

**Review by SlimaneAmar — APPROVED** [review]

**Review by laurgarn — APPROVED** [review]

**Pull request merged**